app/templates/components/navbar.html

```diff
@@ -4,7 +4,26 @@
     <div class="nav-item connection-indicator">
       <connection-indicator id="connection-indicator"></connection-indicator>
     </div>
-
+    <div class="nav-item dropdown">
+      <button class="dropbtn">Cursor</button>
+      <div class="dropdown-content">
+        <a href="#" onclick="return setCursor(this);" class="cursor_default">Default</a>
+        <a href="#" onclick="return setCursor(this);" class="cursor_none">None</a>
+        <a href="#" onclick="return setCursor(this);" class="cursor_crosshair">Crosshair</a>
+        <a href="#" onclick="return setCursor(this);" class="cursor_dot">Dot</a>
+        <a href="#" onclick="return setCursor(this);" class="cursor_pointer">Pointer</a>
+        <a href="#" onclick="return setCursor(this);" class="cursor_cell">Cell</a>
+      </div>
+    </div>
+    <div class="nav-item dropdown">
+      <button class="dropbtn">Screen</button>
+      <div class="dropdown-content">
+        <a href="#" onclick="return setScreen(this);" class="screen_default">Default</a>
+        <a href="#" onclick="return setScreen(this);" class="screen_fill">Fill</a>
+        <a href="#" onclick="return setScreen(this);" class="screen_fillfull"> Fill Fullscreen</a>
```

Comment:
I'd rather treat "Fill Fullscreen" as what we call "Fullscreen." Enabling fullscreen without using the full screen real estate doesn't seem that useful, so I'd rather drop that option.

Reply:
I agree, but there will always be that one person that will ask for that option. I will remove it.